docs/REST API/rest-api.md

```diff
@@ -2511,6 +2511,10 @@ Returns the information of the specified host.
         "coordinates": [40.6799, -74.0028]
       }
     },
+    "maintenance_window": {
+      "starts_at": "2024-08-19T02:02:17Z",
```

Comment:
per conversations, this will now be returned in the user's local time including UTC offset, e.g. `"2024-06-18T13:27:18−07:00 UTC−07:00"`